File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/model/Cast.java
package com.ryanrvldo.moviecatalogue.data.source.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.Data;

@Data
public class Cast {
    @Expose
    private int id;

    @Expose
    private String name;

    @Expose
    private String character;

    @Expose
    private int gender;

    @SerializedName("credit_id")
    @Expose
    private String creditId;

    @SerializedName("profile_path")
    @Expose
    private String profilePath;
}

File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/model/Cast.kt
package com.ryanrvldo.moviecatalogue.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Cast(
    @Expose val id: Int,
    @Expose val name: String,
    @Expose val character: String,
    @Expose val gender: Int,
    @SerializedName("credit_id") @Expose val creditId: String,
    @SerializedName("profile_path") @Expose val profilePath: String
)